src/main/scala/beam/agentsim/scheduler/BeamAgentScheduler.scala

```diff
@@ -76,16 +84,17 @@ object BeamAgentScheduler {
   }
 
   def SchedulerProps(
-    beamConfig: BeamConfig,
-    stopTick: Double = 3600.0 * 24.0,
-    maxWindow: Double = 1.0
-  ): Props = {
-    Props(classOf[BeamAgentScheduler], beamConfig, stopTick, maxWindow)
+                      beamConfig: BeamConfig,
+                      stopTick: Double = 3600.0 * 24.0,
```

Comment:
Is it a unit of time like number of seconds in a day?
some questions:
- why double?
- put the unit of the time in the variable
- use something more meaningful like `TimeUnit.DAYS.toSeconds(1)`